app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb

```diff
@@ -55,12 +55,21 @@ def allowed_to_revert?
     current_state == 'off'
   end
 
+  def revert_to_snapshot_denied_message(active = false)
```

Comment:
@durandom the 'active' is passed from the UI call, when the information about the snapshot is available. See:

https://github.com/ManageIQ/manageiq-ui-classic/pull/845/files#diff-d424af3c1ddd0fc71914bd5b85559c9fR10